src/server.rs

```diff
@@ -398,7 +392,7 @@ impl<S, M> Handler for Server<S, M> where S: Store, M: StateMachine {
 
 impl <S, M> fmt::Debug for Server<S, M> where S: Store, M: StateMachine {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        write!(fmt, "Server({})", self.id)
+        fmt::Debug::fmt(&self.id, fmt)
```

Comment:
I think I would prefer to keep it the way it was.  My understanding is that with this change the debug form of a `Server` would be `'ServerId(42)'`, whereas it is currently `'Server(42)'`?